src/test/java/TurnUseCases/TestMovePlayerUseCase.java

```diff
@@ -0,0 +1,61 @@
+package TurnUseCases;
```

Comment:
it might be a smart idea to add a test for when the player just lands on the jail tile not the go to jail tile